google/cloud/bigtable/app_profile_config.h

```diff
@@ -51,12 +51,13 @@ class AppProfileConfig {
   }
 
   // NOLINT: accessors can (and should) be snake_case.
-  google::bigtable::admin::v2::CreateAppProfileRequest const& as_proto() const {
+  google::bigtable::admin::v2::CreateAppProfileRequest const& as_proto()
+      const& {
     return proto_;
   }
 
   // NOLINT: accessors can (and should) be snake_case.
-  google::bigtable::admin::v2::CreateAppProfileRequest as_proto_move() {
+  google::bigtable::admin::v2::CreateAppProfileRequest&& as_proto() && {
```

Comment:
FWIW, I realize we're omitting the overloads for non-const lvalue and const rvalue. These sometimes make sense in a world of generics where it's important to perfectly preserve the type category. But practically speaking, I think the two overloads you have here are the only ones we really need. 

Just noting this here for posterity.